source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,71 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the ``protectConnectionStrings`` option to hide credentials when 
+users connect to |compass-short| with a connection string. When 
+``protectConnectionStrings`` is enabled, users cannot edit hidden credentials 
+in the |compass-short| connection form, copy the connection string in the 
+|compass-short| interface, or see the credentials when using the :ref:`Export 
+to Language <compass-export-query>` functionality. 
+
+Procedure
+---------
+
+To protect your connection string credentials, enable the 
+``protectConnectionStrings`` option. You can set this option in the 
+|compass-short| :guilabel:`Settings` panel, on the command line or in a 
+:ref:`configuration file <config-file>`. 
+
+Command Line Example
+~~~~~~~~~~~~~~~~~~~~
+
+The following command starts |compass| from the command line and sets
```

Comment:
[nit]

Up until now, we've used the shortened |compass-short| replacement throughout this page. Consider using |compass-short| here as well, and also maybe updating the _first_ instance of Compass on this page to be the longer |compass| replacement.

Reply:
Ah good catch. Updated.